### PR TITLE
Resolve / re-add execution environment list to Organizations

### PR DESCRIPTION
**Pull request by jakemcdermott**
This is actually from https://github.com/ansible/awx/pull/8337

I just rebased, picked the commit, resolved the conflicts. I believe this should do it.

### CODE REVIEW

File: awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.test.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import { mountWithContexts } from '../../../../testUtils/enzymeHelpers';
+
+import OrganizationExecEnvListItem from './OrganizationExecEnvListItem';
+
+describe('<OrganizationExecEnvListItem/>', () => {
+  let wrapper;
+  const executionEnvironment = {
+    id: 1,
+    image: 'https://registry.com/r/image/manifest',
```
Comment (nixocio) [inline awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.test.jsx:12]: do we need to add `name` and `pull` as well?

File: awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvList.jsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+          hasContentLoading={isLoading}
+          items={executionEnvironments}
+          itemCount={executionEnvironmentsCount}
+          pluralizedItemName={i18n._(t`Execution Environments`)}
+          qsConfig={QS_CONFIG}
+          toolbarSearchableKeys={searchableKeys}
+          toolbarRelatedSearchableKeys={relatedSearchableKeys}
+          toolbarSearchColumns={[
+            {
+              name: i18n._(t`Image`),
```
Comment (nixocio) [inline awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvList.jsx:88]: Add `name` as part of the search as well?

File: awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvList.jsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+import useRequest from '../../../util/useRequest';
+import PaginatedDataList from '../../../components/PaginatedDataList';
+import DatalistToolbar from '../../../components/DataListToolbar';
+
+import OrganizationExecEnvListItem from './OrganizationExecEnvListItem';
+
+const QS_CONFIG = getQSConfig('organizations', {
+  page: 1,
+  page_size: 20,
+  order_by: 'image',
```
Comment (nixocio) [inline awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvList.jsx]:
```suggestion
  order_by: 'name',
```
Just to be consistent with the EE main list.

File: awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.jsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      key={executionEnvironment.id}
+      aria-labelledby={labelId}
+      id={`${executionEnvironment.id} `}
+    >
+      <DataListItemRow>
+        <DataListItemCells
+          dataListCells={[
+            <DataListCell
+              key="image"
+              aria-label={i18n._(t`Execution environment image`)}
```
Comment (nixocio) [inline awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.jsx]: Add EE `name` as well?

**Review by nixocio — APPROVED** [review]
Thanks @jakemcdermott, I left minor suggestions that we can add later or on this PR.

File: awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.test.jsx (hunk 1 above)
Comment (jakemcdermott) [reply awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.test.jsx:12]: Good idea - done

File: awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvList.jsx (hunk 2 above)
Comment (jakemcdermott) [reply awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvList.jsx:88]: it is done

**Review by rebeccahhh — APPROVED** [review]
good catch :)

**Review by shanemcd — APPROVED** [review]
LGTM if tests pass!

**Pull request merged**